pkg/perf/udp_task.go

```diff
@@ -0,0 +1,50 @@
+package perf
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/zos/pkg/network/iperf"
+)
+
+// UDPTask is the task for iperf udp tests
+type UDPTask struct {
```

Comment:
This should become one with the other Test because u run both tcp and udp against each selected node and aggreate all results in one decent structure